app/assets/stylesheets/preact/onboarding-modal.scss

```
@@ -1,307 +1,256 @@
+    padding: 16px;
   }
+}
 
-  p {
-    line-height: 1.25em;
-  }
+// Modal Selection Status
+.onboarding-selection-status {
+  align-self: flex-end;
+  flex-shrink: 0;
+  width: 100%;
+  border-top: 2px solid black;
 
-  button {
-    width: 130px;
-    padding: 6px 0px;
-    height: auto;
-    margin: 11px 20px;
+  .selection-status-content {
+    display: block;
+    padding: 16px;
     text-align: center;
-    font-weight: bold;
-    border-radius: 3px;
-    border: 2px solid #0a0a0a;
-    @include themeable(border-color, theme-color, #0a0a0a);
-    @include themeable(color, theme-color, #0a0a0a);
-    @include themeable(
-      background-color,
-      theme-container-accent-background,
-      $light-gray
-    );
-    font-family: 'HelveticaNeue-CondensedBold', 'HelveticaNeueBoldCondensed',
-      'HelveticaNeue-Bold-Condensed', 'Helvetica Neue Bold Condensed',
-      'HelveticaNeueBold', 'HelveticaNeue-Bold', 'Helvetica Neue Bold',
-      'HelveticaNeue', 'Helvetica Neue', 'TeXGyreHerosCnBold', 'Helvetica',
-      'Tahoma', 'Geneva', 'Arial Narrow', 'Arial', sans-serif;
-    -webkit-appearance: none;
-    font-stretch: condensed;
-    font-size: 26px;
-  }
-
-  .next-button {
-    @include themeable(background, theme-top-bar-write-background, #66e2d5);
-    float: right;
   }
+}
 
-  .about {
-    width: 100%;
-
-    label,
-    input,
-    textarea {
-      display: block;
-      width: 95%;
-      margin: 0 auto;
-    }
+// Button Styles
+.onboarding-body button {
+  -webkit-appearance: none;
+  background-color: #3B49DF;
+  border-radius: 3px;
+  border: none;
+  box-sizing: border-box;
+  color: white;
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 150%;
+  padding: 0.5em 1.5em;
+  text-align: center;
+}
 
-    input,
-    textarea {
-      padding: 12px;
-      font-size: 19px;
-      text-emphasis: bold;
-      border: 1px solid #dee6e9;
-      border-radius: 3px;
-    }
+// Tag Styles
+.tag-container {
+  box-sizing: border-box;
+  column-gap: 1em;
+  display: grid;
+  grid-gap: 1em;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: auto;
+  width: 100%;
 
-    textarea {
-      height: 120px;
-    }
-  }
+  .tag-item {
+    background-color: #EEF0F1; // smoke-10
+    border: 2px solid #EEF0F1;
+    border-radius: 3px;
+    display: grid;
+    row-gap: 1em;
+    height: 120px;
+    padding: 16px;
 
-  .checkbox-slide {
-    label {
-      display: block;
+    &:hover {
+      border-color: black;
+      background: white;
       cursor: pointer;
+
+      .tag-item-selector {
+        display: block;
+      }
     }
 
-    input[type='checkbox'] {
-      -webkit-appearance: none;
-      width: 19px;
-      height: 19px;
-      background: transparent;
-      border-radius: 5px;
-      border: 2px solid #555;
-      vertical-align: middle;
-      margin: 8px;
-      cursor: pointer;
+    .tag-topic-name {
+      font-family: 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo,
+      Courier, monospace;
```

There's an existing variable to be used:
```suggestion
      font-family: $monospace;
```